templates/fedora.tpl.yaml

```diff
@@ -17,6 +15,16 @@ metadata:
     template.openshift.io/bindable: "false"
     template.kubevirt.io/version: v1alpha1
     defaults.template.kubevirt.io/disk: rootdisk
+{% if item.workload != "desktop" %}
+    template.kubevirt.io/containerdisks: |
```

Comment:
@dominikholler I think we want to use our containerdisks for all workloads don't we?

Reply:
@yaacov Do you prefer to show no example url for deskop VMs, or an url pointing to the server version?

Reply:
Let's use the server containerdisks, since we have no desktop containerdisks available.

Reply:
Done